docs/getting_started_main.rst

```diff
@@ -3,18 +3,29 @@
 Getting Started
 ===============
 
-What is this?
--------------
+==============================================
+Welcome to the PythonRobotics Documentation 👋
+==============================================
+
+Welcome to PythonRobotics's comprehensive documentation, an open source collection of tried-and-tested robotics algorithms designed to simplify and expedite your robotics journey. 
+PythonRobotics aims to make robot development an enjoyable experience by easing the tedious process of algorithm creation.
+
 
-This is an Open Source Software (OSS) project: PythonRobotics, which is a Python code collection of robotics algorithms.
+Introduction to PythonRobotics
+-------------------------------
 
-The focus of the project is on autonomous navigation, and the goal is for beginners in robotics to understand the basic ideas behind each algorithm.
+PythonRobotics isn't just about cutting down your effort, it's about understanding and learning. 
+It's designed to be a useful resource for budding robotics enthusiasts looking to grasp the fundamental concepts behind each algorithm.
+With its intuitive animations, PythonRobotics simplifies complex concepts, making simulations easily understandable in real-world applications.
 
-In this project, the algorithms which are practical and widely used in both academia and industry are selected.
 
-Each sample code is written in Python3 and only depends on some standard modules for readability and ease of use. 
+Based on the principles of collaboration and knowledge sharing, PythonRobotics is an open source software (OSS). 
+This project includes a selection of practical algorithms that are widely adopted across academic and industrial domains, keeping you on the cutting edge of robotics development.
```

Comment:
Is it possible to describe about minimum dependency policy?

Reply:
ok i'm correcting

Reply:
Here also please explain a little bit what exactly I have to describe.

Reply:
Describing like this chapter
![image](https://github.com/AtsushiSakai/PythonRobotics/assets/3813847/35558874-87ac-4b12-991f-e6081e98a029)
(Ref: https://arxiv.org/pdf/1808.10703.pdf )
is fine.

Reply:
Doing